include/tvm/target/target.h

```diff
@@ -203,5 +204,59 @@ void CheckAndUpdateHostConsistency(Map<Integer, Target>* target, Target* host);
  * \param host The Target typed object for target host to be updated
  */
 void CheckAndUpdateHostConsistency(Map<Target, IRModule>* target, Target* host);
+
+// TODO(@electriclilies): Move to somewhere in backend and add note about appropriate use
```

Comment:
Hey what about moving these methods temporarily to `src/relay/backend/utils.h` instead? Given these are only used in relay backend right now, I think it would be helpful to sort of prevent future developers to use them :-)

Reply:
removed!